app/views/provider_interface/provider_relationship_permissions/_provider_permissions.html.erb

```diff
@@ -0,0 +1,13 @@
+<p class="govuk-body"><%= provider.name %> can:</p>
+<ul class="govuk-list">
+  <% if permissions.view_safeguarding_information %>
+    <li>
+      <svg class="app-icon govuk-!-margin-right-1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 200 200" focusable="false" fill="currentColor" aria-hidden="true">
+        <path d="M100 200a100 100 0 1 1 0-200 100 100 0 0 1 0 200zm-60-85l40 40 80-80-20-20-60 60-20-20-20 20z"></path>
+      </svg>
+      see safeguarding information
+    </li>
+  <% else %>
+    <li>View applications</li>
```

Comment:
Should this appear in both lists, and have a check mark? It looks a bit like the org with permissions granted might not be able to view applications:

![Screenshot 2020-06-02 at 10 13 37](https://user-images.githubusercontent.com/642279/83502652-d359bc80-a4b9-11ea-82c0-459666ac7e07.png)

Reply:
I see there's a design card dedicated to this screen — maybe we should leave it for now?

Reply:
Good point, I wondered about this, it would be nice to have this in all lists as it's not an either/or thing. Will fix. :+1:

Reply:
@duncanjbrown as this was missing the SVG icon I amended the last commit to always display 'view applications' as a permission

![image](https://user-images.githubusercontent.com/93511/83511357-6ef12a00-a4c6-11ea-8c11-22cedc77a1e8.png)

Reply:
This looks great!